src/app/container/container.component.html

```diff
@@ -68,19 +68,15 @@ <h3 id="tool-path">
   <div class="col-md-5">
     <app-starring [tool]="tool" class="pull-right starring-button" *ngIf="isToolPublic && tool" (change)="starGazersChange()"></app-starring>
     <h3 class="pull-right">
-      <div class="button contact-button" *ngIf="publicPage && tool?.private_access">
-        <p>
-          <a [href]="requestAccessHREF">
-            Request Access
-          </a>
-        </p>
+      <div class="contact-button" *ngIf="publicPage && tool?.private_access">
+        <a mat-raised-button color="accent" [href]="requestAccessHREF">
+          Request Access
+        </a>
       </div>
-      <div class="button contact-button" *ngIf="publicPage && tool?.email && !tool?.private_access">
-        <p>
-          <a [href]="contactAuthorHREF">
-            Contact Author
-          </a>
-        </p>
+      <div class="contact-button" *ngIf="publicPage && tool?.email && !tool?.private_access">
```

Comment:
how does it look without contact-button?

Reply:
Hm contact button is just margin really, I'll use a b4 class.